armi/runLog.py

```diff
@@ -528,6 +543,19 @@ def setVerbosity(self, intLevel):
         self.setLevel(intLevel)
 
 
+class NullLogger(RunLogger):
+    def __init__(self, name, isStderr=False):
+        super().__init__(name)
```

Comment:
Typically we call the parent `__init__` directly in ARMI, so `RunLogger.__init__(name)`